lib/saml/user_attributes/ssoe.rb

```diff
@@ -323,41 +321,23 @@ def mhv_iens
         mhv_iens.append(safe_attr('va_eauth_mhvuuid')).reject(&:nil?).uniq
       end
 
-      def mhv_inbound_outbound
+      def mhv_outbound_redirect
```

Comment:
This might be worth memoizing. We call this multiple times and I don't see any reason why we would need to evaluate it each time.
```suggestion
      def mhv_outbound_redirect
        @mhv_outbound_redirect ||= begin
          tracker = SAMLRequestTracker.find(@tracker_uuid)
          tracker&.payload_attr(:skip_dupe_id_validation) == 'true'
        end
      end
```